src/go/rpk/pkg/cli/cmd/start.go

```diff
@@ -169,6 +169,12 @@ func NewStartCommand(fs afero.Fs, mgr config.Manager) *cobra.Command {
 				return err
 			}
 
+			err = mgr.Write(conf)
```

Comment:
I think we should have more info on what this fixes in the commit message.

Reply:
agreed, not sure *why* we have to write the config before starting redpanda. Is this a 2 commit thing?

Reply:
I updated the comment. Thanks!

Reply:
Can you amend it to the commit too? :pray:

Reply:
Done. Sorry!